result/isisiihs.yaml

```diff
@@ -0,0 +1,376 @@
+      description: >-
+        This object contains the status of the last successful Graceful Restart status of a neighbor router.
+      type: object
+      properties:
+        adjacency_bring_up_time:
+          description: |-
+            The time (in second) is taken to bring up adjacency by the Restarting Neighbor.
+            The time is counted by a Helper router when the Restart TLV with RR bit set in a IIH PDU is received 
+            and till it receives the Restart TLV with RR bit unset in a IIH PDU from the Neighbor Router.
+          type: integer
+          format: uint32
+          x-field-uid: 1
+
+    IsisIIH.NeighborGRLastAttemptStatus:
+      description: |-
+        This object contains the status of the last attempted Graceful Restart status of a ISIS neighbor.
+        - succeeded: Corresponding the 'succeeded' object is populated if the last Graceful Status is successful.
+        - failed: Corresponding the 'failed' object is populated if the last Graceful Status is unsuccessful.
+        - unavailable: This value is set if Graceful Restart status is not available at this moment beause either of Graceful
+        Restart status is not given or Graceful Restart status is in progress.
+      type: object
+      properties:
+        choice:
+          type: string
+          x-field-uid: 1
+          x-enum:
+            succeeded:
+              x-field-uid: 1
+            failed:
+              x-field-uid: 2
+            unavailable:
+              x-field-uid: 3
+        succeeded:
+          $ref: '#/components/schemas/IsisIIH.NeighborGRLastAttemptSucceeded'
+          x-field-uid: 2
+        failed:
+          $ref: '#/components/schemas/IsisIIH.NeighborGRLastAttemptFailed'
+          x-field-uid: 3
+        unavailable:
+          type: boolean
+          x-field-uid: 4
+
+    IsisIIH.NeighborGRLastAttemptFailed:
+      description: >-
+        This object contains failure status of Graceful Restart in the last attempted.
+      type: object
+      properties:
+        reason:
+          description: |-
+            Failure reason of last Graceful Restart.
+          type: string
+          x-field-uid: 1
+
+    IsisIIH.NeighborGRLastAttemptSucceeded:
+      description: >-
+        This object contains the result of a successful graceful restart status in the last attempted by a Neighbor.
+      type: object
+      properties:
+        adjacency_bring_up_time:
+          description: |-
+            The time (in second) is taken to bring up adjacency by the Restarting Neighbor.
+            The time is counted by a Helper router when the Restart TLV with RR bit set in a IIH PDU is received 
+            and till it receives the Restart TLV with RR bit unset in a IIH PDU from the Neighbor Router.
+          type: integer
+          format: uint32
+          x-field-uid: 1
+
+
+
+    IsisIIH.Tlvs:
+      description: >-
+        This contains the list of TLVs present in a IIH PDU.
```

Comment:
Again , for future proofing , might be better to keep this different for local vs remote e.g. probably tester would be much more interested in what is received from DUT and more TLVs might be needed in future whereas for what we send , even if needed it might be a small subset .